Loop/Managers/ExtensionDataManager.swift

```diff
@@ -118,6 +118,8 @@ final class ExtensionDataManager {
         #endif
 
         context.lastLoopCompleted = loopDataManager.lastLoopCompleted
+        context.lastCGMComm = loopDataManager.mostRecentGlucoseDataDate
```

Comment:
nit: the name indicates the last time of *comms*, but the value tracked is actually the date of the most recent data. May cause a future dev to make an incorrect assumption that the value is really the last comms date.

Reply:
Gotcha, so it's possible that there's a comm from the pump that would not be "good" data? Is `mostRecentXDataDate` a good indicator of having no device issues?

Reply:
Okay, naming updated.